manager/controllers/app/plotter_controller.go

```diff
@@ -30,6 +30,12 @@ import (
 	"fybrik.io/fybrik/pkg/multicluster"
 )
 
+const (
+	plotterLiteral           = "Plotter"
+	noRemoteBlueprintWarnMsg = "Could not yet find remote blueprint"
+	plotterReadyMsg          = "Plotter is ready!"
```

Comment:
I don't think messages should be defined as constants.

Reply:
Should I change them to vars instead or to not save them at all?

Reply:
Not save them at all. Inside the code assign a string variable and reuse it for both conditions and log messages.